--- java/com/google/tinkuser/TinkUser.java
+++ java/com/google/tinkuser/TinkUser.java
@@ -19,4 +19,5 @@
 import com.google.crypto.tink.Aead;
 import com.google.crypto.tink.KeysetHandle;
+import com.google.crypto.tink.TinkProtoKeysetFormat;
 import java.security.GeneralSecurityException;
 
@@ -24,5 +25,5 @@
 public final class TinkUser {
   public Aead useReadNoSecret(byte[] b) throws GeneralSecurityException {
-    return KeysetHandle.readNoSecret(b).getPrimitive(Aead.class);
+    return TinkProtoKeysetFormat.parseKeysetWithoutSecret(b).getPrimitive(Aead.class);
   }
 }
